com.unity.render-pipelines.high-definition/Documentation~/Master-Node-Decal.md

```diff
@@ -62,9 +62,26 @@ To view these properties, click the **Cog** in the top right of the Master Node.
 
 ### Material Properties
 
-These properties are in the **Exposed Properties** section of the Inspector, alongside the properties that you exposed in the Shader Graph's Blackboard. If you set **Override ShaderGUI** to `true`, the Material Properties section does not appear, and instead, the ShaderGUI you specified appears.
+When creating a Decal Material from the Decal MAster node, following properties are available, alongside the properties that you exposed in the Shader Graph's Blackboard.
 
-| Property                  | Description                                                  |
+#### Surface Options
+
+These properties allow you to set the affected attributes of the Material the decal is project onto when HDRP renders it into the Scene.
+
+| **Property**                    | **Description**                                              |
+| ------------------------------- | ------------------------------------------------------------ |
+| **Affect BaseColor**            | Enable the checkbox to make this decal use the **baseColor** properties. Otherwise the decal has no baseColor effect. Either disabled or enabled, the alpha channel of the base color is still use as an opacity for its other properties. |
```

Comment:
Regardless of whether you enable or disable this property, HDRP still uses the alpha channel of the base color as an opacity for the other properties.